### PR TITLE
Fix various MDA 2.0 introduced deprecations (removal for 3.0)

### PR DESCRIPTION
**Pull request by IAlibay**

### CODE REVIEW

File: doc/source/groups_of_atoms.rst [hunk 1]
```diff
@@ -208,4 +208,4 @@ and see which fragments are associated with atoms in a smaller :class:`~MDAnalys
 
 .. note::
 
-    :attr:`AtomGroup.fragments <MDAnalysis.core.groups.AtomGroup.fragments>` returns a tuple of fragments with at least one :class:`~MDAnalysis.core.groups.Atom` in the :class:`~MDAnalysis.core.groups.AtomGroup`, not a tuple of fragments where *all* Atoms are in the :class:`~MDAnalysis.core.groups.AtomGroup`.
\ No newline at end of file
+    :attr:`AtomGroup.fragments <MDAnalysis.core.groups.AtomGroup.fragments>` returns a tuple of fragments with at least one :class:`~MDAnalysis.core.groups.Atom` in the :class:`~MDAnalysis.core.groups.AtomGroup`, not a tuple of fragments where *all* Atoms are in the :class:`~MDAnalysis.core.groups.AtomGroup`.
```
Comment (lilyminium) [inline doc/source/groups_of_atoms.rst]: I might be blind, but i can't see any change here :/

**Review by lilyminium — APPROVED** [review]
LGTM thanks!

File: doc/source/groups_of_atoms.rst (hunk 1 above)
Comment (IAlibay) [reply doc/source/groups_of_atoms.rst]: Odd that wasn't a file I touched. Maybe an EOL introduction?

Comment (IAlibay) [reply doc/source/groups_of_atoms.rst]: Yeah looks like empty lines added, github's editor can't remove them either. If it's not breaking things I'm tempted to just say let's keep them?

**Pull request merged**